charts/tekton-pipeline/README.md

```diff
@@ -36,6 +36,9 @@ helm upgrade --install --set auth.docker.username=fred --set auth.docker.passwor
 
 | Parameter                           | Description                                             | Default                                                                                     |
 |------------------------------------|---------------------------------------------------------|---------------------------------------------------------------------------------------------|
+| `crds.install`                   | Whether to install CRDs (Custom Resource Definitions)  | `true`                                                                          |
```

Comment:
You forgot the value in the README. I think it would be good to point out the crd handling elsewhere in the README